content/300-guides/200-deployment/201-serverless/500-deploy-to-netlify.mdx

```diff
@@ -6,190 +6,24 @@ metaDescription: 'Learn how to deploy Node.js and TypeScript applications that a
 
 <TopBlock>
 
-This guide shows how to deploy a serverless Node.js application that uses Prisma to [Netlify](https://www.netlify.com/). The example below exposes a REST API and uses Prisma Client to fetch, create, and delete database records.
+This guide shows how to deploy a Node.js application that uses Prisma to [Netlify](https://www.netlify.com/).
 
 Netlify is a cloud platform for continuous deployment, static sites, and serverless functions. Netlify integrates seamlessly with GitHub for automatic deployments upon commits. When you follow the steps below, you will use that approach to create a CI/CD pipeline that deploys your application from a GitHub repository.
 
-The application has the following components:
-
-- **Backend**: Serverless Node.js REST API with resource endpoints that use Prisma Client to handle database operations against a PostgreSQL database (e.g. hosted on Heroku).
-- **Frontend**: Static HTML page to interact with the API.
-
-![Prisma Netlify GitHub architecture diagram](images/500-01-deploy-to-netlify-architecture.png)
-
-The focus of this guide is to show how Prisma integrates with Netlify. The starting point is the [Prisma Netlify example](https://github.com/prisma/deployment-example-netlify) that has a couple of REST endpoints preconfigured as serverless functions and a static HTML page.
-
 With Netlify, the fundamental building block is a [**Site**](https://docs.netlify.com/configure-builds/get-started/). Sites are typically connected to a Git repository and have a publicly accessible URL that looks like `https://site-name.netlify.app`. In this guide, you will connect a GitHub repository to a Netlify site.
 
 </TopBlock>
 
 ## Prerequisites
 
-- a GitHub account
-- a hosted PostgreSQL database and a connection string for it, such as `postgresql://username:password@your_postgres_db.cloud.com/db_identifier` (you can use Supabase, which offers a [free plan](https://dev.to/prisma/set-up-a-free-postgresql-database-on-supabase-to-use-with-prisma-3pk6)).
-- a [Netlify](https://www.netlify.com/) account connected to your GitHub account (Netlify needs access to the repository you will create as part of this guide).
+- An existing project using Prisma (this example uses a Next.js application, but can be applied to any application using Prisma)
+- A [Netlify](https://www.netlify.com/) account connected to your GitHub account (Netlify needs access to the repository you will deploy as part of this guide).
```

Comment:
Is this really the only way to deploy to Netlify? If not, we might want to expand this guide into showing the different options later.

Reply:
_theoretically_ this is the [the comprehensive list of options](https://docs.netlify.com/welcome/add-new-site/).

I think it would be wise to cover more, but some of those flows are low on my priority list

Reply:
"Deploy local files" is the one that is the AWS CLI equivalent but is quite handy.

Reply:
Created [#5141](https://github.com/prisma/docs/issues/5141)